memcache_proxy/dtog_test.go

```diff
@@ -749,6 +749,7 @@ func TestAll(t *testing.T) {
 				"STAT curr_items 444\r\n",
 				"STAT bytes 555\r\n",
 				"STAT oldest_item_age 666\r\n",
+				"STAT version not implemented\r\n",
```

Comment:
We settled on "App Engine", no? That's what I see on HEAD.

On Sep 8, 2016 6:43 PM, "Brent Shaffer" <notifications@github.com> wrote:

> In memcache_proxy/dtog_test.go
> <https://github.com/GoogleCloudPlatform/appengine-sidecars-docker/pull/28#discussion_r78118165>
> :
>
> > @@ -749,6 +749,7 @@ func TestAll(t *testing.T) {
> >  				"STAT curr_items 444\r\n",
> >  				"STAT bytes 555\r\n",
> >  				"STAT oldest_item_age 666\r\n",
> > +				"STAT version not implemented\r\n",
>
> we decided on "not implemented" last time, I think we should stick with
> that. I can turn it into a constant though, I like that 👍
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/appengine-sidecars-docker/pull/28/files/b47f38a6a850dba709da0f60b2d184f6b657dbce#r78118165>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABhlgsxyclxScwXZ5sc9WPlfah8Kxoyks5qoLm5gaJpZM4Jufz6>
> .
>